templates/shared/forms/form-template.html

```diff
@@ -0,0 +1,50 @@
+{% if isModal %}
+  <div class="p-modal" id="{{ modalId }}">
```

Comment:
```suggestion
  <div class="p-modal js-modal-ready" id="{{ modalId }}">
```
You need this js-hook so we can check the modal has loaded in dynamicForm.js